File: TransCoder_tokenized_test_set_functions/python/GIVEN_TWO_NUMBERS_B_FIND_X_X_B.py
def modularEquation ( a, b ) :
    if ( a < b ) :
        print ( "No solution possible " )
        return
    if ( a == b ) :
        print ( "Infinite Solution possible " )
        return
    count = 0
    n = a - b
    y = ( int ) ( math . sqrt ( a - b ) )
    for i in range ( 1, y + 1 ) :
        if ( n % i == 0 ) :
            if ( n / i > b ) :
                count = count + 1
            if ( i > b ) :
                count = count + 1
    if ( y * y == n and y > b ) :
        count = count - 1
    print ( count )

File: TransCoder_tokenized_test_set_functions/java/GIVEN_TWO_NUMBERS_B_FIND_X_X_B.java
static void modularEquation ( int a, int b ) {
  if ( a < b ) {
    System . out . println ( "No solution possible " );
    return;
  }
  if ( a == b ) {
    System . out . println ( "Infinite Solution possible " );
    return;
  }
  int count = 0;
  int n = a - b;
  int y = ( int ) Math . sqrt ( a - b );
  for ( int i = 1;
  i <= y;
  ++ i ) {
    if ( n % i == 0 ) {
      if ( n / i > b ) count ++;
      if ( i > b ) count ++;
    }
  }
  if ( y * y == n && y > b ) count --;
  System . out . println ( count );
}